src/lib/functions/runtimes/js/builders/zisi.ts

```diff
@@ -76,18 +87,22 @@ const buildFunction = async ({
   } = await memoizedBuild({
     cache,
     cacheKey: `zisi-${entryPath}`,
-    // @ts-expect-error TS(2345) FIXME: Argument of type '{ archiveFormat: string; basePat... Remove this comment to see the full error message
-    command: () => zipFunction(entryPath, targetDirectory, zipOptions),
+    command: async () => {
+      const result = await zipFunction(entryPath, targetDirectory, zipOptions)
+      if (result == null) {
+        throw new Error('Failed to build function')
+      }
```

Comment:
A lot of code would have been a (worse) nightmare to type without this, so this seems worth including in this PR